packages/runtime/src/execute/expression.ts

```diff
@@ -100,13 +101,30 @@ export const wrapOperation = (
   fn: Operation,
   logger: Logger,
   name: string,
-  immutableState?: boolean
+  immutableState?: boolean,
+  prevName?: string
```

Comment:
@josephjclark 
This was needed to know the name of the previous operation.

So we can say. `The previous operation ${prevName} didn't return a state. did you forget?`.
should the wording include the previous name?